microservices/case-service/pom.xml

```diff
@@ -70,6 +71,24 @@
 			<groupId>software.amazon.awssdk</groupId>
 			<artifactId>s3</artifactId>
 		</dependency>
+		<!-- Support for Java8+ datatype marshalling -->
+		<dependency>
+			<groupId>com.fasterxml.jackson.datatype</groupId>
+			<artifactId>jackson-datatype-jsr310</artifactId>
+			<version>2.13.3</version>
```

Comment:
I know it's strange, I had to add the version again to make the service start from the command prompt, otherwise I get:

[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.springframework.boot:spring-boot-starter-data-jpa:jar -> duplicate declaration of version (?) @ line 56, column 15
[ERROR] 'dependencies.dependency.version' for com.amazonaws:aws-java-sdk-s3:jar is missing. @ line 65, column 15

The Intellij IDE reports the error only after syncing the POM (but I always value the result from the terminal over the build on the IDE)

Reply:
Also check duplicate `spring-boot-starter-data-jpa` definition